projects/batfish/src/org/batfish/grammar/cisco/Cisco_bgp.g4

```diff
@@ -501,10 +523,11 @@ null_bgp_tail
       | NEIGHBOR_DOWN
       | PASSWORD
       | SEND_LABEL
+      | SHUTDOWN
       | SOFT_RECONFIGURATION
       | SUPPRESS_FIB_PENDING
       | SYNCHRONIZATION
-      | TIMERS
+      | timers_bgp
```

Comment:
First, see earlier comment about not putting parser rules in null_bgp_tail. Second, this change (along with the definition of timers_bgp) is completely non-functional and should be reverted.